docs/legacy/sourcemap-api.asciidoc

```diff
@@ -0,0 +1,88 @@
+[[sourcemap-api]]
+== Source map upload API
```

Comment:
I don't think this is used anymore, and users should be using the k/ibana sourcemap upload API these days.

Was this meant to be added here?

Reply:
whoops, nope this was accidental. I'll remove

Reply:
I think this was a conflict that I resolved incorrectly